File: Uri Online Judge/PHP/1116.php
<?php

$quantidadePares = intval(readline());

for ($i = 0; $i < $quantidadePares; $i++) { 
    list($valor1, $valor2) = array_map(fn($x) => intval($x), explode(' ', readline()));
    if ($valor2 == 0) {
        echo 'divisao impossivel' . PHP_EOL;
        continue;
    }
    printf("%.1f" . PHP_EOL, $valor1 / $valor2);
}

File: Uri Online Judge/Python3/1116.py
quantidade_pares = int(input())

for i in range(quantidade_pares):
    valores = input().split()
    if int(valores[1]) == 0:
        print('divisao impossivel')
        continue
    print('{0:.1f}'.format(float(valores[0])/float(valores[1])))
